docs/ShippingSimulator.md

```diff
@@ -7,52 +7,52 @@ Shipping Simulator is a VTEX component that estimates the shipping fee based on
-| `shippingTable` | The table for the shipping methods | [ShippingTable](/react/components/ShippingSimulator/components/ShippingTable.js) |
-| `shippingTableCell` | The cell of a table | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
-| `shippingTableLabel` | The label describing a shipping method | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
-| `shippingTableRadioBtn` | The radio button to select a shipping method | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
\ No newline at end of file
+| Class name                   | Description                                                | Component Source                                                                       |
+| ---------------------------- | ---------------------------------------------------------- | -------------------------------------------------------------------------------------- |
+| `shippingContainer`          | The main container of the `shippingCalculator`             | [index](/react/components/ShippingSimulator/index.js)                                  |
+| `shippingContainerLoader`    | The container for the spinner loader                       | [index](/react/components/ShippingSimulator/index.js)                                  |
+| `shippingZipcodeLabelLoader` | The container for the ZIP code loader                      | [index](/react/components/ShippingSimulator/index.js)                                  |
+| `shippingInputLoader`        | The container for the ZIP code input loader                | [index](/react/components/ShippingSimulator/index.js)                                  |
+| `shippingZipcodeLabel`       | The shipping label                                         | [index](/react/components/ShippingSimulator/index.js)                                  |
+| `shippingNoMessage`          | The message when the store does not have a shipping method | [ShippingTable](/react/components/ShippingSimulator/components/ShippingTable.js)       |
+| `shippingTable`              | The table for the shipping methods                         | [ShippingTable](/react/components/ShippingSimulator/components/ShippingTable.js)       |
+| `shippingTableCell`          | The cell of a table                                        | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
+| `shippingTableLabel`         | The label describing a shipping method                     | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
+| `shippingTableRadioBtn`      | The radio button to select a shipping method               | [ShippingTableRow](/react/components/ShippingSimulator/components/ShippingTableRow.js) |
```

Comment:
Add this text below the handles: 
"In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization). "